VogtPayroll5/Payroll.cs

```diff
@@ -1,34 +1,90 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.IO;
 
 namespace VogtPayroll5
 {
     class Payroll
     {
+        //total number of hours worked by all employees, and total gross pay as well
+        private static int countHours;
+        private static decimal totalGrossPay;
+
         private List<Employee> _empList = new List<Employee>();
+
         public void Run()
         {
-            List<Employee> empList = new List<Employee>();
             PayrollConsoleReader payrollConsoleReader = new PayrollConsoleReader();
```

Comment:
This payroll class shouldn't be calling the console reader directly like this. Instead, you should be building the employee **outside** of the payroll class and have an `AddEmployee` function which will push the employee to the `_empList` array.

Reply:
Really this feels like it should be your Program.cs. It's the highest logical flow happening. 1. Read from console, 2. calculate totals, 3. write to file